app/src/main/java/com/beemdevelopment/aegis/importers/DatabaseImporter.java

```diff
@@ -40,6 +40,7 @@
         _appImporters.put("FreeOTP", FreeOtpImporter.class);
         _appImporters.put("Google Authenticator", GoogleAuthImporter.class);
         _appImporters.put("Steam", SteamImporter.class);
+        _appImporters.put("Authy", AuthyImporter.class);
```

Comment:
Let's keep these lists sorted alphabetically. Can you also add it to the list of regular importers?